Segments/Bloodlands.mapproj

```diff
@@ -96,7 +96,7 @@
 
         public override void OnSpellTarget(Target target, MobileEntity combatant)
         {
-            if (target != null)
+            if (target = null)
```

Comment:
Should this be == not =? Single ='s are normally used to assign a value where as == are equality checks.

Reply:
Ugh yes I think you’re correct needs ==

Reply:
Are you able to make that correction ? Won’t be at a computer for another few hours

Reply:
It won't allow editing on github because of the file size and I'd have to open a completely new pull for the change. Probably best to wait until you can update it.